backend/data/static/create_master_files.py

```diff
@@ -0,0 +1,115 @@
+import pandas as pd
+import os
+from functools import reduce
+
+# Set up directory for the os
+directory = "cleaned/"
+
+# Initialise some values for the loop
+la = {}
+i = 0
+
+# Read in all the dataframes to a dictionary
+for filename in os.listdir(directory):
+    if filename.startswith("LA_"):
+        la["df{}".format(i)] = pd.read_csv("cleaned/" + filename, index_col=[0, 1])
+        i += 1  # Add one to the i constant
+        continue
+    else:
+        continue
+
+# Make a list of all the dataframes for the merge.
+la_list = list(la.values())
+# Merge all the dataframes on the indexes
+la_master = reduce(
+    lambda left, right: pd.merge(left, right, left_index=True, right_index=True),
+    la_list,
+)
+
+# Now do the same for the lsoa!
+# Initialise some values for the loop
+lsoa = {}
+j = 0
+
+# Read in all the dataframes to a dictionary
+for filename in os.listdir(directory):
+    if filename.startswith("LSOA_"):
+        lsoa["df{}".format(j)] = pd.read_csv("cleaned/" + filename, index_col=[0, 1])
+        j += 1  # Add one to the i constant
+        continue
+    else:
+        continue
```

Comment:
AH AH... look at this Code clone! 

Soooo, if you now make that a function, and you **just** change the value of the parameter `filename_prefix`... isn't

Reply:
Perhaps renaming this function from `collect_las` to  `collect_authorities_data` maybe?